src/de.rs

```
@@ -669,7 +632,13 @@ impl<'de> de::Deserializer<'de> for QsDeserializer<'de> {
     {
         let s = match self.0 {
             ParsedValue::String(s) => s,
-            ParsedValue::Sequence(mut seq) => get_last_string_value(&mut seq)?,
+            ParsedValue::Sequence(mut seq) => {
+                if let Some(v) = get_last_string_value(&mut seq) {
```

I am not sure about this approach, is it common to represent an string as an array? 

I suppose this might be when you have the value specified multiple times but you are only asking for a singular value?

Is it desired to silently ignore extra values?

---

> I suppose this might be when you have the value specified multiple times but you are only asking for a singular value?

Yup this is exactly is. This is to match the use case in #68 which is a fairly common web querystring use case. We could make this be configurable thogh.

> On another note, this function mutates the array, so calling it removes the last element regardless if it returns Some or None. So if we then forward the value to the visit_seq below it seems that we would be missing the last value.

Good catch!